### ISSUE TITLE
Add-Migration fails when AudioRoutingEndpoint is used within UWP App

### ISSUE THREAD

**Issue by RonPenton** [dotnet/efcore#7008]
### Steps to reproduce
I have a UWP app which includes a function in a Xaml.cs file that references "AudioRoutingEndpoint". Whenever I try to perform an "Add-Migration" command, the execution fails with an exception stating that it cannot find this type. It should be noted that the use of this type occurs nowhere on any of the model or context classes; it's purely in the UI and has nothing to do with the Entity Framework. 

When I comment out the reference to this type, the migrations begin working again. 

### The issue
```
Exception message: System.TypeLoadException: Could not find Windows Runtime type 'Windows.Phone.Media.Devices.AudioRoutingEndpoint'.
Stack trace:
`System.TypeLoadException: Could not find Windows Runtime type 'Windows.Phone.Media.Devices.AudioRoutingEndpoint'.
   at System.RuntimeTypeHandle.GetDeclaringType(RuntimeType type)
   at System.RuntimeType.RuntimeTypeCache.GetEnclosingType()
   at System.RuntimeType.get_DeclaringType()
   at System.RuntimeType.RuntimeTypeCache.GetNameSpace()
   at System.RuntimeType.get_Namespace()
   at Microsoft.EntityFrameworkCore.Migrations.Design.MigrationsScaffolder.<>c__DisplayClass17_0.<ContainsForeignMigrations>b__0(TypeInfo t)
   at System.Linq.Enumerable.<>c__DisplayClass6_0`1.<CombinePredicates>b__0(TSource x)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source)
   at Microsoft.EntityFrameworkCore.Migrations.Design.MigrationsScaffolder.ContainsForeignMigrations(String migrationsNamespace)
   at Microsoft.EntityFrameworkCore.Migrations.Design.MigrationsScaffolder.ScaffoldMigration(String migrationName, String rootNamespace, String subNamespace)
   at Microsoft.EntityFrameworkCore.Design.Internal.MigrationsOperations.AddMigration(String name, String outputDir, String contextType)
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.AddMigrationImpl(String name, String outputDir, String contextType)
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.AddMigration.<>c__DisplayClass0_1.<.ctor>b__0()
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.<>c__DisplayClass3_0`1.<Execute>b__0()
   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.Execute(Action action)
Could not find Windows Runtime type 'Windows.Phone.Media.Devices.AudioRoutingEndpoint'.`
```

### Further technical details

EF Core version: 1.0.1
Operating system: Windows 10, latest update.
Visual Studio version: Visual Studio 2015 Community, Update 3

**Comment by bricelam** [comment]
@RonPenton Could you share a repro project to help us investigate? (If not, I'll do my best without it.)

**Comment by bricelam** [comment]
I am not able to reproduce this. I suspect it may be a duplicate of #4683 which was fixed in 1.0.0-preview3-final of the tools.
